File: Ejercicio6.py


def asignar_grupo():
    nombre = input("Ingresa tu nombre: ")
    genero = input("Ingresa tu género (Masculino/Femenino/Otro): ").lower()

    if genero == "masculino":
        grupo = "Grupo A"
    elif genero == "femenino":
        grupo = "Grupo B"
    else:
        grupo = "Grupo C"

    print(f"{nombre}, has sido asignado al {grupo}.")

if __name__ == "__main__":
    asignar_grupo()

File: Ejercicio6.hs
module Ejercicio6 where

import System.IO



asignarGrupo :: String -> String -> String
asignarGrupo nombre genero
    | genero == "Masculino" = "Grupo A"
    | genero == "Femenino"  = "Grupo B"
    | otherwise             = "Grupo C"

main :: IO ()
main = do
    hSetBuffering stdout NoBuffering
    putStr "Ingrese el nombre: "
    nombre <- getLine
    putStr "Ingrese el género (Masculino/Femenino/Otro): "
    genero <- getLine
    let resultado = asignarGrupo nombre genero
    putStrLn $ "El grupo asignado es: " ++ resultado


